### ISSUE TITLE
Tasks misspelled in querying guide

### ISSUE THREAD

**Issue by edwardloveall** [luckyframework/website#92]
https://luckyframework.org/guides/database/querying#associations-and-joins

```crystal
UserQuery.new.join_taks.tasks { |task_query|
  # WHERE tasks.title = 'Clean up notes'
  task_query.title("Clean up notes")
}
```

`join_taks` should be `join_tasks`

**Comment by jwoertink** [comment]
This will be fixed by #93

**Comment by jwoertink** [comment]
Fixed and merged by https://github.com/luckyframework/website/pull/99.
